CHANGELOG.md

```diff
@@ -65,6 +65,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 * Fixed relative path to the current directory in output to be `.` instead of
   empty string.
 
+* When adding a new workspace, the parent of the current workspace's commit
```

Comment:
"parent of the current workspace's commit" is a bit difficult to understand, perhaps because I'm relatively new to the project. Perhaps "parent of the current workspace's current checkout" or something?

Reply:
Good point. Done.